server/streamreader/jack_stream.cpp

```diff
@@ -156,8 +156,13 @@ JackStream::JackStream(PcmStream::Listener* pcmListener, boost::asio::io_context
             throw SnapException("JackStreams only support 16, 24 and 32 bit sample formats");
     }
 
+#if 0
+    // Disabled by default as aixlog <= 1.4.0 is not thread-safe at the moment
+    // and Jack produces quite a few log message.
+    // TODO: enable if and when aixlog becomes thread-safe.
```

Comment:
What means not thread safe? As a stream based logger, it might mix log messages, but it should not crash.
What does Jack log? Doesn't it make sense to log at least the errors?

Reply:
Sorry, I don't mean "unsafe" as in crashing. But the "meta-information" of log messages get mixed up if multiple threads access the logger at the same time. See also https://github.com/badaix/aixlog/issues/14. As Jack outputs bursts of a few messages, it becomes quite noticeable.